### DISCUSSION TITLE
Are there any Fork Projects currently in progress?

### DISCUSSION THREAD

**Discussion by Lorde627** [m-bain/whisperX#1057] [General]
It seems like m-bian has given up on this project; he doesn’t even deal with pull requests, but is secretly contributing to his private projects. I was wondering if there’s a future fork of this project?

**Comment by NADOOITChristophBa** [comment]
I am working on a Metal fork that runs fast on Mac.

https://github.com/NADOOIT/whisperX

**Comment by TinaTe** [comment]
@NADOOITChristophBa  How's that going so far? I did check it out, but I barely see any changes, so I was wondering.

**Comment by NADOOITChristophBa** [comment]
I haven't gotten to it in the last couple of days. We are currently moving to decentralized Android. That way, we can use all our Phones and any other hardware as a communal AI Cluster. Currently got like 300 GHz and couple hundred GB of VRAM going ;)

The metal support was moving along nicely. Most of that is in the cTranslate2 fork of mine.
I think I was working on WhisperX to add a couple features like auto fine-tuning and individualized speech models so that the accuracy should be close to 100%. Also was supposedly multiple times real time so the model would actually fine-tune as you use it.

It could be that is all on the cTranslate2 as well.

I hope to add a couple people onto this but at the moment I am the only Mac User in our group

https://github.com/NADOOIT/CTranslate2

**Comment by TinaTe** [comment]
I fear that making use of that is far beyond my capabilities :D 
But thanks for taking the time!

**Comment by remic33** [comment]
I am trying to update the versions of pyannote and faster-whisper but for now, I am on a dead end. 
If we do not upgrade Pyannote, we do not have access to numpy 2+, and the library becomes basically useless.

**Comment by NADOOITChristophBa** [comment]
As far as I can tell, this is still the best open-source project for this. My company is growing, so I will add a couple of people to this next year. We will be moving to mostly Android next year, so I wonder how well this will work there. I do not understand how there is not more focus on this. Am I missing any other projects that have surpassed this? Speech-to-text clearly is the future. I mean, most people who are done with school can talk and listen fine, but they cannot read or write.
